### DISCUSSION TITLE
Axis Doorstation A8105 with Fritzbox

### DISCUSSION THREAD

**Discussion by RyanMirror** [AxisCommunications/discussions#251] [General]
Hello all,

I would like to connect the said doorstation to my fritz.box via sip and if possible also via live video. With my existing Doorbirds this had worked perfectly. Only the Axis does not like. I have entered everything, but the connection does not work.
Can you help, please?

Greetings
Adrian

**Comment by vivekatoffice** [comment]
Related link:
[AXIS A8105-E – Direct calling to 2N® Indoor Touch 2.0](https://wiki.2n.com/faqint/en/axis-a8105-e-prime-volani-na-2n-indoor-touch-2-0-104863762.html)
